themes/default/content/docs/guides/saml/aad.md

```diff
@@ -113,13 +113,13 @@ download the SAML application's configuration data and then pass that to Pulumi.
 ## Signing into Pulumi using Azure AD
 
 Once your Azure AD application is created, and its configuration data passed to Pulumi, you can now
-sign in to the Pulumi Service using your SAML SSO credentials.
+sign in to the Pulumi Cloud using your SAML SSO credentials.
```

Comment:
```suggestion
sign in to the Pulumi Cloud console using your SAML SSO credentials.
```

Reply:
@jkodroff lets leave this as it was before this pr opened, we can come back later if we establish a standard around the.